hack/verify-spelling.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
we add this to check all files

`golangci-lint` just checks the go source code